examples/kubectl_wrapper_example/main.tf

```
@@ -20,7 +20,7 @@ locals {
 
 module "enabled_google_apis" {
   source  = "terraform-google-modules/project-factory/google//modules/project_services"
-  version = "~> 10.2"
+  version = "~> 13"
```

nit: Lets pin to a minor version
```suggestion
  version = "~> 13.0"
```

---

Thanks, didn't catch that when I updated the module version.